onnxruntime/core/providers/xnnpack/detail/utils.h

```diff
@@ -21,6 +21,7 @@ namespace onnxruntime {
 class GraphViewer;
 class NodeUnit;
 namespace xnnpack {
+constexpr const char* kDynamicDomainByCreate = "xnnpack.create.domain";
```

Comment:
nit: kXnnpackDomain = "xnnpack" would be fine. how that is used (dynamic creation or some other way in the future) doesn't need to be in the name or value.